resources/views/roles/edit-add.blade.php

```diff
@@ -67,7 +67,7 @@
                                         <ul>
                                             @foreach($permission as $perm)
                                                 <li>
-                                                    <input type="checkbox" id="permission-{{$perm->id}}" name="permissions[]" class="the-permission" value="{{$perm->id}}" @if(in_array($perm->key, $role_permissions)) checked @endif>
+                                                    <input type="checkbox" id="permission-{{$perm->id}}" name="permissions[{{$perm->id}}]" class="the-permission" value="{{$perm->id}}" @if(in_array($perm->key, $role_permissions)) checked @endif>
```

Comment:
Is there a particular reason this was added? Form arrays in Laravel are identified by `<fieldname>[]`.  No ID value necessary.

Reply:
It doesn't change anything saving but I didn't find any simple way to check or uncheck it in tests without a unique name, referencing id doesn't work.